### PR TITLE
Made 5 astroids spawn at a random location.

### PR DESCRIPTION
**Pull request by Ashthecool**
5 asteroids now start at a random location

### CODE REVIEW

File: my_game.py [hunk 1]
```diff
@@ -316,11 +318,12 @@ def setup(self):
...
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
+            new_asteroid = Asteroid(
+            center_x=0,
```
Comment (tobiasbp) [inline my_game.py:323]: Asteroid should not accept center_x and center_y as inputs anymore.

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 1 above)
Comment (Ashthecool) [reply my_game.py:323]: Jeg ved ikke hvorfor den gør det. Jeg gætter på det er, fordi vi fjernede x=0 og y=0 oppe ved Astroids

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py [hunk 2]
```diff
@@ -316,11 +318,9 @@ def setup(self):
...
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
+            new_asteroid = Asteroid()
+            self.asteroid_list.append(new_asteroid)
```
Comment (tobiasbp) [inline my_game.py:323]: No need fot the variable new_asteroid. Let's create the Asteroid() directly in the .append()

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 2 above)
Comment (Ashthecool) [reply my_game.py:323]: Oh alright, I understand

File: my_game.py [hunk 3]
```diff
@@ -78,7 +78,9 @@ def __init__(self, center_x, center_y):
             filename="images/Meteors/meteorGrey_big1.png", 
             scale=SPRITE_SCALING
         )
-        
+
+        center_x = random.randint(0, SCREEN_WIDTH)
+        center_y = random.randint(0, SCREEN_HEIGHT)
```
Comment (tobiasbp) [inline my_game.py:83]: Please set self.center_x and self.center_y directly. That way, you don't need the following two lines.

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 3 above)
Comment (Ashthecool) [reply my_game.py:83]: Got it

File: my_game.py [hunk 4]
```diff
@@ -316,11 +316,8 @@ def setup(self):
         )
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
```
Comment (tobiasbp) [inline my_game.py:319]: Let's put the number of Asteroids in a global variable `ASTEROIDS_PR_LEVEL = 5`

**Review by tobiasbp — CHANGES_REQUESTED** [review]

File: my_game.py (hunk 4 above)
Comment (Ashthecool) [reply my_game.py:319]: Got it! :)

Comment (tobiasbp) [reply my_game.py]: You forgot the the S in ASTEROIDS

Comment (Ashthecool) [reply my_game.py]: Nu er der et S

File: my_game.py [hunk 5]
```diff
@@ -315,12 +316,9 @@ def setup(self):
...
         )
         
-        # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        # Spawn Asteroids
+        for r in range(ASTROIDS_PR_LEVEL):
```
Comment (tobiasbp) [inline my_game.py:320]: ASTEROIDS.... Not ASTROIDS

**Review by tobiasbp — CHANGES_REQUESTED** [review]

**Review by tobiasbp — APPROVED** [review]

**Pull request merged**